loadtest_manifest.yaml

```diff
@@ -22,4 +26,15 @@ gateways:
   - key_name: gateway2
     exposed_url: http://localhost:42080
   - key_name: gateway3
-    exposed_url: http://localhost:42081
\ No newline at end of file
+    exposed_url: http://localhost:42081
+
+# Persistent chain manifest example:
```

Comment:
Why can't we enable this in this PR?

Reply:
Since the current manifest is used to run the test. The `TestNet` and `LocalNet` scenarios are mutually exclusive, so only one of them could be uncommented/activated.

As mentioned in previous comments, I suggest having a second manifest that exercises testing non-ephemeral chains. We'll indicate which one to trigger by pointing to the corresponding manifest file.

WDYT?